ansible/roles/udb/templates/prologin/roots.list

```diff
@@ -0,0 +1 @@
+admin:admin
```

Comment:
This should be in the ansible inventory, unless you want everyone to be able to root the infra with `admin:admin`